Puppetfile

```diff
@@ -23,11 +23,14 @@ mod 'puppetlabs-zone_core', '1.0.2'
 
 # Useful additional modules
 mod 'puppetlabs-package', '0.6.0'
-mod 'puppetlabs-puppet_conf', '0.3.1'
+mod 'puppetlabs-puppet_conf', '0.4.0'
 mod 'puppetlabs-python_task_helper', '0.3.0'
 mod 'puppetlabs-reboot', '2.2.0'
 mod 'puppetlabs-ruby_task_helper', '0.4.0'
 
+# Plugin modules
+#mod 'puppetlabs-azure_inventory', '0.2.0'
```

Comment:
@nicklewis We need to figure out why the push to forge job is failing. https://jenkins-master-prod-1.delivery.puppetlabs.net/job/forge-module_puppetlabs-azure_inventory_init-tag_push-forge/3/console

Reply:
Alex pointed out that it is not that important to publish this module to the forge. Pulling from git tag for now.